lib/node_modules/@stdlib/stats/base/svariancewd/manifest.json

```
@@ -72,7 +79,26 @@
         "-lm"
       ],
       "libpath": [],
-      "dependencies": []
+      "dependencies": [
+        "@stdlib/blas/base/shared"
+      ]
+    },
+    {
+      "task": "",
+      "wasm": true,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [
+        "-lm"
+      ],
```

```suggestion
      "libraries": [],
```
we don't need this anymore, same for all other instances in this file